src/_posts/2020-02-20-remote-pairing-a-true-story.md

```diff
@@ -0,0 +1,68 @@
+
+But, how does this last statement matches with the context presented in the paragraph above?
+
+## Pairing Remote is still pairing
+Pair Programming is a demanding activity and remote pairing is even more demanding, but it doesn't need to be even more exhausting, while it also can be done effectively. Here there is a compound of lessons learned that might help your team boost their pair programming experience:
+
+- Requires good and interrupted connections
+- Get comfortable headset for your team, avoid microphones that capture too much environmental noise.
+- There are dozens of tools available, but stick to a video conference that allows only one driver.
+- Ask questions and be prepared to explain, keep your pair engaged
+- Is not that simple to say "here" or "there", use the proper method or functions names, use the line numbers to reference a location in the file.
+- When information is lacking, use your video tool of reference to invite an external and evacuate doubts
+- Eventually you can try mob programming (for me didn't work, but I know many of my colleagues had great experiences doing mob remotely)
+
+The rest is basically use the common sense and the general pairing guidelines. I add some suggestions that might help you throughout your pairing experience:
+- Involve the whole team when doing planning meetings, make them feel part and organize the meetings taking remotes into account.
+- Have a refined backlog of work, which the whole involved team took part in its creation.
+- Share context, make them part of the discussions and own the topic (this is obviously beyond Pair Programming but is IMHO a key factor)
+- Shift the pairs and balance skills, as despite not being a mentoring activity, it won't work if none of the pair members has experience on the topic or technology.
+
+We are craftspeople, we like to solve complex problems in the best way possible working in the best way engineering practices encourage. In the hyperconnected context were live, definitely we can find a proper way to still work remotely as effectively as if we are together.
+
+## Conclusion
+Pair programming is though, and requires practice and some sort of expertise from the involved people. If you want some maturity on the team. There are many strategies and suggested approaches to tackle such activity.
+
+There are an innumerable amount of developers that object the benefits of it without even trying it. Trust me, try it, give it a run and evaluate the results. You'll be tired but the outcome of your craft will be beneficial for the whole team and you'll also learn from your pair.
+
+And do not allow remote working prevent you from doing it, above I left some hints based on our experience working in a Codurance project where, despite its ups and downs, we managed to work collaboratively as a team with the involvement of Barcelona, London and Manchester team members.
```

Comment:
I would move - 'And do not allow remote working to prevent you from doing it.' to the end of the paragraph above'

And I would word this paragraph like - 

> In this article I have provided some hints based on my experience working on a Codurance project where, despite its ups and downs, we worked collaboratively as a team with the involvement of my colleagues across our Barcelona, London and Manchester locations and I hope they help you in your pairing experiences in the future.